Swiftfin/Views/ItemView/iPadOS/ScrollViews/iPadOSCinematicScrollView.swift

```diff
@@ -0,0 +1,165 @@
+                }
+            }
+            .frame(height: UIScreen.main.bounds.height * 0.8)
+        }
+
+        var body: some View {
+            ScrollView(showsIndicators: false) {
+                VStack(spacing: 0) {
+                    VStack(spacing: 0) {
+                        Spacer()
+
+                        OverlayView(viewModel: viewModel)
+                            .padding2(.horizontal)
+                            .padding2(.bottom)
+                    }
+                    .frame(height: UIScreen.main.bounds.height * 0.8)
+                    .background {
+                        BlurView(style: .systemThinMaterialDark)
+                            .mask {
+                                LinearGradient(
+                                    stops: [
+                                        .init(color: .clear, location: 0.4),
+                                        .init(color: .white, location: 0.8),
+                                    ],
+                                    startPoint: .top,
+                                    endPoint: .bottom
+                                )
+                            }
+                    }
+                    .overlay {
+                        Color.systemBackground
+                            .opacity(topOpacity)
+                    }
+
+                    content()
+                        .padding(.vertical)
+                        .background(Color.systemBackground)
+                }
+            }
+            .edgesIgnoringSafeArea(.top)
+            .edgesIgnoringSafeArea(.horizontal)
+            .scrollViewOffset($scrollViewOffset)
+            .navBarOffset(
+                $scrollViewOffset,
+                start: UIScreen.main.bounds.height * 0.65,
+                end: UIScreen.main.bounds.height * 0.65 + 50
+            )
+            .backgroundParallaxHeader(
+                $scrollViewOffset,
+                height: UIScreen.main.bounds.height * 0.8,
+                multiplier: 0.3
+            ) {
+                headerView
+            }
+        }
+    }
+}
+
+extension ItemView.iPadOSCinematicScrollView {
+
+    struct OverlayView: View {
+
+        @EnvironmentObject
+        private var itemRouter: ItemCoordinator.Router
+        @ObservedObject
+        var viewModel: ItemViewModel
+
+        var body: some View {
+            VStack(alignment: .leading) {
+                ImageView(
+                    viewModel.item.imageURL(.logo, maxWidth: 500),
+                    resizingMode: .aspectFit
+                ) {
+                    Text(viewModel.item.displayName)
+                        .font(.largeTitle)
+                        .fontWeight(.semibold)
+                        .lineLimit(2)
+                        .multilineTextAlignment(.leading)
+                        .foregroundColor(.white)
+                }
+                .frame(maxWidth: UIScreen.main.bounds.width * 0.4, maxHeight: 100)
```

Comment:
It looks really cool with the logo, but it looks like there is an alignment problem with the text only.
It seems that it needs to be solved by adjusting the alignment of the frame, not by adjusting the text alignment.
```swift
.frame(maxWidth: UIScreen.main.bounds.width * 0.4, maxHeight: 100, alignment: .leading)
```

There may be a similar problem in other places that you I haven't checked.

Reply:
Please see the item about the `ImageView` in "🧠 dump". This issue affects both the logo and the text.

Reply:
Oh, I can understand that part now.

Then my opinion on this is

In the `iPadOS CinematicScrollView.OverlayView`, I think that normally work if the modifier only be applied to FailureView.

However, In other situations, it may be necessary to use the modifier of `FailureView`, which is contrary to the modifier of `ImageView`, so it would be good to provide the initializer mentioned in ImageView at the same time.

For implementation in the way you mentioned, it would be good to refer to Nuke's `ImageContainer`.

Reply:
Oh, I can understand that part now.

Then my opinion on this is

In the `iPadOS CinematicScrollView.OverlayView`, I think that normally work if the modifier only be applied to `FailureView`.

However, In other situations, it may be necessary to use the modifier of `FailureView`, which is contrary to the modifier of `ImageView`, so it would be good to provide the initializer in ImageView as you said.

However, there will be another problem because of the logic that should be handled in common with ImageView. (Placeholder you mentioned)

we need a better structure...
I think it would be good to think about it together.

Reply:
It also does _not_ work if the modifier is applied to the `FailureView`.

Reply:
Oh, it only works if the `alignment` is added to _both_ the failure view and the image view. For now, I kinda want to hold off on adding those modifiers due to the `ImageView` change I present which would these problems.

Reply:
Yes, I got it.
I will also think about `ImageView` refactoring.